src/pages/guidelines/color/overview.mdx

```diff
@@ -149,21 +160,28 @@ color.
 <br />
 <Title> Global background colors </Title>
 
-| Theme    | Primary background      | Token            | Hex value |                                             |
-| -------- | ----------------------- | ---------------- | --------- | ------------------------------------------: |
-| White    | Global Background Light | `$ui-background` | `#ffffff` |  <ColorBlock size="xs">#ffffff</ColorBlock> |
-| Gray 10  | Global Background Light | `$ui-background` | `#f4f4f4` | <ColorBlock  size="xs">#f4f4f4</ColorBlock> |
-| Gray 90  | Global Background Dark  | `$ui-background` | `#262626` | <ColorBlock  size="xs">#282828</ColorBlock> |
-| Gray 100 | Global Background Dark  | `$ui-background` | `#161616` | <ColorBlock  size="xs">#171717</ColorBlock> |
+| Theme    | Primary background      | Token         | Hex value |                                             |
+| -------- | ----------------------- | ------------- | --------- | ------------------------------------------: |
+| White    | Global Background Light | `$background` | `#ffffff` |  <ColorBlock size="xs">#ffffff</ColorBlock> |
+| Gray 10  | Global Background Light | `$background` | `#f4f4f4` | <ColorBlock  size="xs">#f4f4f4</ColorBlock> |
```

Comment:
When looking at the preview, this might need a tweak in the component since the border isn't strong enough to differentiate between the background and the color for our pages, I think.